rootbld makes all the supplementary groups nobody(65534);
this patch avoids using them in tests

diff --git a/getgroups.c b/getgroups.c
index 8a37ed0..01adfca 100644
--- a/getgroups.c
+++ b/getgroups.c
@@ -48,6 +48,7 @@
 #endif
 
 	for (i = 0; i < n; i++)  {
+		if (list[i] == 65534) continue;
 		printf("%lu ", (unsigned long)list[i]);
 		if (list[i] == gid)
 			gid_in_list = 1;
